assets/javascripts/modules/autoComplete.js

```diff
@@ -415,8 +413,9 @@ var setup = function ($elem, suggestionsData, $targetInputElem, suggestionFormat
     * If this is not supplied then the autoComplete input will contain the
     * suggestion.title which will be sent to the backend as the value of the $autoCompleteInputElem
     */
-    $targetInput = $targetInputElem || getTargetInput($elem);
-
+    $targetInputs = getTargetInput($elem) || $targetInputElem;
+    $targetInputs = $targetInputs.split(" ");
```

Comment:
what are you expecting $targetInputs to be here? With the work currently on master `getTargetInput($elem)` returns an element and `$targetInputElem` is an element.